layouts/partials/header.html

```diff
@@ -30,6 +30,13 @@ <h3 class="subhead-main">A peer-to-peer hypermedia protocol<br /> designed to ma
           <a class="button button-primary" href="#get-started">Get started</a>
           <a class="button button-outlined" href="#how">How it works</a>
         </div>
+        <div class="hero-callout">
+          <div class="hero-callout-image"><img src="/images/05-rocket.svg" /></div>
+          <div class="hero-callout-text">
+            <h3><strong>It's here! Try go-ipfs 0.5 now</strong></h3>
+            <p>Learn all about it in the <a href="">blog post</a>, check out the <a href="">docs</a>, or <a href="">install go-ipfs 0.5</a>.</p>
```

Comment:
```suggestion
            <p>Learn all about it in the <a href="https://blog.ipfs.io/">blog post</a>, check out the <a href="https://docs-beta.ipfs.io/recent-releases/go-ipfs-0-5/">docs</a>, or <a href="https://dist.ipfs.io/#go-ipfs">install go-ipfs 0.5</a>.</p>
```
Adds links to correct docs and install pages. Blog page is just generic; will update specific once that's live